src/qt/bitcoin.cpp

```diff
@@ -450,6 +450,16 @@ WId BitcoinApplication::getMainWinId() const
     return window->winId();
 }
 
+bool BitcoinApplication::event(QEvent* e)
+{
+    if (e->type() == QEvent::Quit) {
```

Comment:
Is there a reason `QEvent::Quit` is undocumented (and apparently unused in anything else Google finds)?

Reply:
I don't know.

Reply:
Would be nice if there's a way we can fix this without undocumented (undefined?) behaviour.

Worst case, partially reverting #336 instead? :/

Reply:
> Is there a reason `QEvent::Quit` is undocumented (and apparently unused in anything else Google finds)?

> Would be nice if there's a way we can fix this without undocumented (undefined?) behaviour.

Qt codease, as every other codebase, is not perfect. And if documentation diverges from code, it is an bug in documentation (https://bugreports.qt.io/browse/QTBUG-92122) which has already been fixed but not backported.

> Worst case, partially reverting #336 instead? :/

Yes, this solution is also possible. Only there are two concerns:
- do we really want to return to undocumented way to interrupt the main event loop during application run?
- is it feasible for 23.0 considering [release schedule](https://github.com/bitcoin/bitcoin/issues/22969)?

Reply:
>it is an bug in documentation (https://bugreports.qt.io/browse/QTBUG-92122) which has already been fixed but not backported.

Ok, seems fine if that's all it is